docs/ROADMAP.md

```diff
@@ -0,0 +1,148 @@
+# Prysmatic Labs Roadmap Reference
+
+This document serves as a main reference for Prysmatic Labs' sharding and beacon chain implementation in Go, along with our roadmap and compilation of active research.
+
+# Table of Contents
+
+-   [Sharding Introduction](#sharding-introduction)
+    -   [Basic Sharding Idea and Design](#basic-sharding-idea-and-design)
+-   [Roadmap Phases](#roadmap-phases)
+    -   [The Ruby Release: Local Network](#the-ruby-release-local-network)
+    -   [The Sapphire Release: Goerli Testnet](#the-sapphire-release-goerli-testnet)
+    -   [The Diamond Release: Ethereum Mainnet](#the-diamond-release-ethereum-mainnet)
+    -   [System Architecture](#system-architecture)
+-   [Acknowledgements](#acknowledgements)
+-   [References](#references)
+
+# Sharding Introduction
+
+Currently, every single node running the Ethereum network has to process every single transaction that goes through the network. This gives the blockchain a high amount of security because of how much validation goes into each block, but at the same time it means that an entire blockchain is only as fast as its individual nodes and not the sum of its parts. Currently, transactions on the EVM are non-parallelizable, and every transaction is executed in sequence globally. The scalability problem then has to do with the idea that a blockchain can have at most 2 of these 3 properties: decentralization, security, and scalability.
```

Comment:
Optional comment: I wish we had some markdown formatter. It would be nice if this block of text was split over several lines.